misc/dist/html/editor.html

```diff
@@ -706,7 +706,7 @@ <h2 id="welcome-modal-title" class="welcome-modal-title">Important - Please read
 		} else if (typeof err === 'string') {
 			setStatusNotice(err);
 		} else {
-			setStatusNotice('An unknown error occured');
+			setStatusNotice('An unknown error occurred');
```

Comment:
```suggestion
			setStatusNotice('An unknown error occurred.');
```
I'm not sure of the context of these messages, but it's possible that they should be sentence case with a period, if we're already updating them. There is another similar instance.